modules/monitoring-reviewing-monitoring-dashboards-developer.adoc

```diff
@@ -6,16 +6,20 @@
 [id="reviewing-monitoring-dashboards-developer_{context}"]
 = Reviewing monitoring dashboards as a developer
 
-In the *Developer* perspective, you can view dashboards relating to a selected project. You must have access to monitor a project to view dashboard information for it.
+In the *Developer* perspective, you can view dashboards relating to a selected project.
+
+[NOTE]
+====
+In the *Developer* perspective, you can view dashboards for only one project at a time.
+====
 
 .Prerequisites
 
-* You have access to the cluster as a developer or as a user.
 * You have view permissions for the project that you are viewing the dashboard for.
```

Comment:
@juzhao what would be the role that gives these permissions? I think it would be useful to mention the role specifically. Is it some of the monitoring roles or are we talking about a different permissions here?

Reply:
@eromanova97 any user who create the project or have view permission for the project can view the project dashboard. maybe we don't need to describe too clearly